### PR TITLE
Fill questions answers 

### PR DESCRIPTION
**Pull request by vlad12345123**

### CODE REVIEW

File: exalearn-api/ExaLearn.Bl/DTO/TopicQuestionDTO.cs [hunk 1]
```diff
@@ -6,8 +6,6 @@ public class TopicQuestionDTO
     {
         public int Id { get; set; }
 
-        public LevelType LevelType { get; set; }
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/DTO/TopicQuestionDTO.cs:9]: when you create topics level not needed?

File: exalearn-api/ExaLearn.Bl/Services/AudioFileService.cs [hunk 2]
```diff
@@ -18,47 +12,19 @@ namespace ExaLearn.Bl.Services
...
-        public AudioFileService(IAudioFileRepository audioFileRepository, IConfiguration configuration, IMapper mapper)
+        public AudioFileService(IAudioFileRepository audioFileRepository, IMapper mapper)
         {
             _audioFileRepository = audioFileRepository;
-            _configuration = configuration;
             _mapper = mapper;
         }
 
-        public async Task<AudioFileDTO> CreateAsync(IFormFile file)
+        public async Task<AudioFileDTO> CreateAsync(AudioFileDTO audioFileDTO)
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/Services/AudioFileService.cs]: for what? question has string url

File: exalearn-api/ExaLearn.Dal/ModelBuilderExtensions.cs [hunk 3]
```diff
@@ -72,6 +72,24 @@ public static void Seed(this ModelBuilder builder)
                 Text = "Goat"
             };
 
+            var topicAnswer1 = new Answer
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/ModelBuilderExtensions.cs]: topics does't containt answers, question containts text

File: exalearn-api/ExaLearn.Bl/DTO/TopicQuestionDTO.cs (hunk 1 above)
Comment (vlad12345123) [reply exalearn-api/ExaLearn.Bl/DTO/TopicQuestionDTO.cs:9]: yes, i talk about this with front team

File: exalearn-api/ExaLearn.Bl/Services/AudioFileService.cs (hunk 2 above)
Comment (vlad12345123) [reply exalearn-api/ExaLearn.Bl/Services/AudioFileService.cs]: agree

Comment (romanstat) [reply exalearn-api/ExaLearn.Bl/Services/AudioFileService.cs]: remove audiofilecontroller

File: exalearn-api/ExaLearn.Dal/ModelBuilderExtensions.cs [hunk 4]
```diff
@@ -85,14 +85,30 @@ public static void Seed(this ModelBuilder builder)
...
                 LevelType = LevelType.Beginner
             };
 
-            builder.Entity<Question>().HasData(grammarQuestion, listeningQuestion);
+            var topic1 = new Question
+            {
+                Id = 3,
+                QuestionType = QuestionType.Topic,
+                Text = "Let’s talk about Great Britain. What do you know about this country?",
+                LevelType = LevelType.Beginner
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/ModelBuilderExtensions.cs]: level type for topic?

File: exalearn-api/ExaLearn.Dal/ModelBuilderExtensions.cs [hunk 5]
```diff
@@ -85,14 +85,30 @@ public static void Seed(this ModelBuilder builder)
...
+            };
+
+            var topic2 = new Question
+            {
+                Id = 4,
+                QuestionType = QuestionType.Topic,
+                Text = "Can you speak in your micro about nature (2 minutes)?",
+                LevelType = LevelType.Beginner
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Dal/ModelBuilderExtensions.cs]: same

**Review by romanstat — APPROVED** [review]

**Review by HenadzV — APPROVED** [review]

**Pull request merged**